features-json/css-content-visibility.json

```
@@ -539,7 +539,7 @@
   },
   "notes":"",
   "notes_by_num":{
-    
+    "1":"Enabled in Firefox by setting the `about:config` preference `layout.css.content-visibility.enabled` to true",
```

I believe the build is complaining about the extra comma, this should work:

```suggestion
    "1":"Enabled in Firefox by setting the `about:config` preference `layout.css.content-visibility.enabled` to true"
```